### PR TITLE
[PWGMM][dndeta-mft] binned in centrality, and added INEL>0 event class

### PR DESCRIPTION
**Pull request by sarahherrmann**
need new dependency : trackselection, centrality-table and multiplicity-table

### CODE REVIEW

File: PWGMM/Mult/TableProducer/trackPropagation.cxx [hunk 1]
```diff
@@ -98,11 +98,10 @@ struct AmbiguousTrackPropagation {
 
   void initCCDB(ExtBCs::iterator const& bc)
   {
-    LOG(info) << "INITIALIZING CCDB";
-
     if (runNumber == bc.runNumber()) {
       return;
     }
+    LOG(info) << "INITIALIZING CCDB";
```
Comment (aalkin) [inline PWGMM/Mult/TableProducer/trackPropagation.cxx:104]: Probably not very useful, since ccdb queries have their own printouts. Additionally, when running on hyperloop you get the statistics of executed queries. In general, it is recommended to avoid non-informative printouts.

File: PWGMM/Mult/Tasks/dndeta-mft.cxx [hunk 2]
```diff
@@ -35,20 +35,35 @@
 using namespace o2;
 using namespace o2::framework;
 using namespace o2::framework::expressions;
+using namespace o2::aod::track;
```
Comment (aalkin) [inline PWGMM/Mult/Tasks/dndeta-mft.cxx:38]: Why is this needed? Further in the code you still use `aod::track::` namespace prefix.

File: PWGMM/Mult/Tasks/dndeta-mft.cxx [hunk 3]
```diff
@@ -143,6 +160,33 @@ struct PseudorapidityDensityMFT {
...
+      registry.add({"Events/Centrality/Selection", ";status;centrality;events", {HistType::kTH2F, {{4, 0.5, 4.5}, CentAxis}}});
+      auto hstat = registry.get<TH2>(HIST("Events/Centrality/Selection"));
+      auto* x = hstat->GetXaxis();
+      x->SetBinLabel(1, "All");
+      x->SetBinLabel(2, "Selected");
+      x->SetBinLabel(3, "Selected INEL>0");
+      x->SetBinLabel(4, "Rejected");
+
+      registry.add({"Events/Centrality/NtrkZvtx", "; N_{trk}; Z_{vtx} (cm); centrality", {HistType::kTH3F, {MultAxis, ZAxis, CentAxis}}});
+      registry.add({"Events/Centrality/NtrkZvtx_gt0", "; N_{trk}; Z_{vtx} (cm); centrality", {HistType::kTH3F, {MultAxis, ZAxis, CentAxis}}});
```
Comment (aalkin) [inline PWGMM/Mult/Tasks/dndeta-mft.cxx:173]: We do not need INEL>0 for Pb-Pb, in my code it is a leftover from copying that I intend to remove.

**Review by aalkin — CHANGES_REQUESTED** [review]
Remove INEL>0 that we do not use for HI.

File: PWGMM/Mult/TableProducer/trackPropagation.cxx (hunk 1 above)
Comment (sarahherrmann) [reply PWGMM/Mult/TableProducer/trackPropagation.cxx:104]: Ok I removed it

File: PWGMM/Mult/Tasks/dndeta-mft.cxx (hunk 2 above)
Comment (sarahherrmann) [reply PWGMM/Mult/Tasks/dndeta-mft.cxx:38]: This is needed for all the definitions of the TrackSelectionFlags, I think it will be a bit too long if I remove it and write aod::track:: everywhere

File: PWGMM/Mult/Tasks/dndeta-mft.cxx (hunk 3 above)
Comment (sarahherrmann) [reply PWGMM/Mult/Tasks/dndeta-mft.cxx:173]: Ok I will remove this

**Review by aalkin — APPROVED** [review]
